_episodes/03-dependencies.md

```diff
@@ -195,42 +195,26 @@ To get an idea of what's needed, let's have a look at the
 
 ---
 
-> ## Exercise: Creating and exporting conda environments
+> ## Exercise: Exploring conda environments
 >
-> On Windows, we recommend to do this exercise in the Anaconda Prompt.
->
-> Begin by **first importing** (by clicking "Use this template")
-> and then cloning the 
-> [word-count project](https://github.com/coderefinery/word-count).
-> Then recreate the software environment provided by the
-> `requirements.txt` file in the repository. 
-> The `snakemake-minimal` package is only available in the `bioconda`
-> channel, which needs to be specified:
-> ```shell
-> $ conda create --name wordcount --file requirements.txt --channel bioconda --channel conda-forge
-> ```
-> This will download all the packages listed in the `requirements.txt`
-> file (with matching versions) along with all dependencies.
-> The new environment also needs to be activated:
-> ```shell
-> $ conda activate wordcount
-> ```
-> We now have (roughly) the same environment as specified by the
-> developers of the word-count project. But let's say we want to
-> extend this environment, and share it with colleagues:
 > - Inspect your available environments with `conda info -e`.
-> - Install the `pandas` package using `conda install`.
-> - Add `pandas` with the version you installed to the `requirements.txt` file.
-> - Export the full environment using `conda env export > full_env.yml`, and 
+> - Activate the base directory with `conda deactivate`.
```

Comment:
this first step is a bit surprising (activate vs deactivate)

Reply:
Good spotting it. I agree it can be revised for clarity. I will deal with it tomorrow.

Reply:
Thank you for pointing out the need for revision. These first steps have been changed.

Reply:
Thanks!